packages/gatsby-source-wordpress/src/utils/fetch-graphql.ts

```diff
@@ -302,37 +327,41 @@ const handleFetchErrors = async ({
       },
     })
   }
-  if (e.message.includes(`Request failed with status code 50`)) {
-    const {
-      requestConcurrency,
-      previewRequestConcurrency,
-    } = store.getState().gatsbyApi.pluginOptions.schema
+  if (
+    e.message.includes(`Request failed with status code 50`) &&
+    [`502`, `503`, `504`].includes(e.message)
+  ) {
+    if (`message` in e) {
+      console.error(formatLogMessage(new Error(e.message).stack))
+    }
 
-    console.error(e)
     reporter.panic({
       id: CODES.WordPress500ishError,
       context: {
         sourceMessage: formatLogMessage(
-          [
-            `Your wordpress server at ${bold(url)} appears to be overloaded.`,
-            `\nTry reducing the ${bold(
-              `requestConcurrency`
-            )} for content updates or the ${bold(
-              `previewRequestConcurrency`
-            )} for previews:`,
-            `\n{
-  resolve: 'gatsby-source-wordpress',
-  options: {
-    schema: {
-      requestConcurrency: 5, // currently set to ${requestConcurrency}
-      previewRequestConcurrency: 2, // currently set to ${previewRequestConcurrency}
-    }
-  },
-}`,
-            `\nThe ${bold(
-              `GATSBY_CONCURRENT_REQUEST`
-            )} environment variable is no longer used to control concurrency.\nIf you were previously using that, you'll need to use the settings above instead.`,
-          ].join(`\n`),
+          `Your WordPress server at ${bold(url)} appears to be overloaded.
+
+${getLowerRequestConcurrencyOptionMessage()}`,
+          { useVerboseStyle: true }
+        ),
+      },
+    })
+  } else if (e.message.includes(`Request failed with status code 500`)) {
+    reporter.panic({
+      id: CODES.WordPress500ishError,
```

Comment:
`500ishError` I love this :joy:

Reply:
haha :laughing: Sam I think you made that code!!